.github/workflows/ci.yml

```diff
@@ -100,6 +100,27 @@ jobs:
       - name: Clippy check
         run: cargo clippy --all-targets --all-features --workspace -- -D warnings
 
+  schema:
+    name: Schema
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@stable
+      - uses: Swatinem/rust-cache@v2
+      - name: Rename committed schema
+        run: mv .schema/latest.json .schema/latest_orig.json
+      - name: Run release-plz generate-schema
+        run: cargo run --package release-plz --bin release-plz --release generate-schema
+      - name: Check schema
+        uses: LouisBrunner/diff-action@v0.2.0
+        with:
+          old: .schema/latest_orig.json
+          new: .schema/latest.json
+          mode: addition
+          tolerance: same
```

Comment:
I was thinking..we could have a cargo test to do this check instead of adding this CI job 🤔

Reply:
Apart from this minor point, this PR looks perfect, thanks 🙏 
I will test this asap.

Reply:
Yeah.. that would have been a lot easier!
I will make the change, plus it should be quicker to run.